content/Foundation/Summit/2024/home.pml

```diff
@@ -80,11 +80,9 @@
 </div>
 
 <div class="summit_div">
-  <div class="highlight">
-    <h3 id="summit-news">
-      Latest News: Program Committee
-    </h3>
-  </div>
+  <h3 id="summit-pc">
+    Latest News: Program Committee
```

Comment:
Remove "Latest news" in the title (I would actually not have "Latest news" in the section headers, just in the top summary, so avoid to much updates in the future)